src/core/typings.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+declare module 'query-string' {
```

Comment:
I didn't want to include global typings.
It seems we are okay to update `query-string` to the version shipped with type definitions. We had to roll back to the old version due to compatibility issues with IE11 https://github.com/elastic/kibana/pull/59633

Reply:
Can't we have a ts project for `/typings` and import the reference?

Reply:
we already have `@kbn/utility-types` for standard types, I'd rather avoid creating another utility package until it's unavoidable